src/WebUI/Pods/PodsLeftPanel.scss

```diff
@@ -4,13 +4,15 @@
     margin-top: 15px;
 
     .pod-left-panel-header {
-        padding-left: 20px;
+        .pod-left-panel-back-button {
+            cursor: pointer;
```

Comment:
people change this to their own way, so when they change do we honor it? for example on a link, by default we show hand, instead if my default is someother cursor type, do we show it when cursor is of pointer type?

Reply:
Cursor pointer means the hand cursor only.
I have set it on back button icon as it is a clickable target, and was not set by default